pkg/storaged/storage.css

```diff
@@ -723,3 +723,11 @@ td.storage-action {
 .sidepanel-row .spinner {
     display: inline-block;
 }
+
+.panel-heading {
+    position: static;
+}
+
+.dropdown-toggle.btn-primary {
+    color: white;
```

Comment:
I'd prefer this to be uniquely classed in the HTML and then you use the class here, instead of telling it that all dropdown-toggles + primary buttons to have white text.

(Also, I'm pretty sure primary buttons have white text by default, so I'm not sure why this is here?)

Reply:
This is indeed not necessary.  There was something that made it go black in the very beginning when focused, I think.  Maybe a bug in Patternfly that got fixed.

Reply:
Removed.